src/app/pages/card-import/card-import.page.ts

```diff
@@ -170,13 +181,23 @@ export class CardImportPage implements OnInit {
 
   updateCard() {
       this.cardService.updateCard(this.card).then(() => {
-      this.router.navigateByUrl('/profile');
+      this.router.navigateByUrl('/home/home/profile');
         this.showToast('Card updated');
       }, err => {
         this.showToast('There was a problem updating your card :(');
       });
   }
 
+  sendCard() {
+     let navigationExtras: NavigationExtras = {
+        state: {
+            cardDataFront: this.card.frontImg,
+            cardDataBack: this.card.backImg
+        }
+     };
+     this.router.navigate(["/home/home/exchange"], navigationExtras);
```

Comment:
Also the global `/exchange` path exists in `app-routing.module.ts`.

Reply:
A "nesting issue" is definitely in order.